HISTORY.rst

```diff
@@ -17,6 +17,10 @@ History
   * ``paysera``
   * ``payvision``
   * ``trustly``
+* Adds support for eight digit ``issuer_id_number`` to ``/credit_card``.
+* Adds support for ``last_digits`` alias of ``last_4_digits`` to ``/credit_card``.
+* Adds support for two digit ``last_digits`` to ``/credit_card``.
+* Deprecates ``last_4_digits`` in favor of ``last_digits``.
```

Comment:
We are inconsistent between different languages, but everything else here uses past tense.